content/2023/06-13-join-box-developer-community.md

```diff
@@ -0,0 +1,41 @@
+---
+applied_at: "2023-06-13"
+applies_to: 
+- api
+- node
+- python
+- java
+- windows
+- cli
+- ui-elements
+- reports
+- salesforce
+- ios
+- android
+- sdks
+is_impactful: false
+is_new_feature: true
+collapse: true
+show_excerpt: true
+release_source_url: ''
+---
+
+# Join Box Developer Community 
+
+👋 Exciting news for developers and admins! Box has launched the Box Developer Community site, where you can connect with peers and experts to build innovative applications on top of the Box cloud content management and collaboration platform. Join us today at https://forum.box.com
+
+<!-- more -->
+
+# Join, connect and learn
+
+* Get support & provide feedback
+* Share and become a community leader
```

Comment:
It seems like something is missing after `Share`. Maybe `Share experiences` or `Share knowledge`?